libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.h

```diff
@@ -25,17 +25,6 @@
 #import <Foundation/Foundation.h>
 
 /**
- @enum Logging levels to control the verbosity of log output based on the severity of the event being logged.
- */
-typedef NS_ENUM(NSUInteger, SFOAuthLogLevel) {
```

Comment:
Not that it should necessarily continue to have its own, but that we should translate its previous log levels into the new paradigm.  The old way we did that was to honor the OAuth-specific check in the form of an inline conditional, then log at the lowest level if that condition was met.  Here, we took out the conditional, but didn't consider its purpose, leaving 'Debug' logs for all of those.

So e.g. for the following code:
```obj-c
if (self.credentials.logLevel < kSFOAuthLogLevelWarning) {
    [self log:SFLogLevelDebug format:@"%@ host=%@ : path=%@", NSStringFromSelector(_cmd), url.host, url.path];
}
```
we should change to:
```obj-c
[SFSDKCoreLogger w:[self class] format:@"%@ host=%@ : path=%@", NSStringFromSelector(_cmd), url.host, url.path];
```